src/overlays/actors/ovl_Bg_Treemouth/z_bg_treemouth.c

```diff
@@ -25,10 +25,10 @@ void func_808BC8B8(BgTreemouth* this, PlayState* play);
 void func_808BC9EC(BgTreemouth* this, PlayState* play);
 void func_808BCAF0(BgTreemouth* this, PlayState* play);
 
-extern CutsceneData D_808BCE20[];
-extern CutsceneData D_808BD2A0[];
-extern CutsceneData D_808BD520[];
-extern CutsceneData D_808BD790[];
+extern CutsceneData gDekuTreeMeetingCs[];
+extern CutsceneData gDekuTreeChoiceCs[];
+extern CutsceneData gDekuTreeChoiceAcceptedCs[];
+extern CutsceneData gDekuTreeChoiceDeclinedCs[];
 
```

Comment:
Feel like these need different names but im not sure what. 
`gDekuTreeChoiceAcceptedCs` should probably be named after the mouth opening
`gDekuTreeChoiceDeclinedCs` is really "coming back to ask again after declining the first time" but I dont know how to say that briefly

Reply:
I chose to these names because it's cutscenes that happens depending on the player's choice in the textbox, for `gDekuTreeChoiceDeclinedCs` maybe "choose again"? and I assume for `gDekuTreeChoiceAcceptedCs` it would be `gDekuTreeMouthOpeningCs`

Reply:
Yeah I understand your original thought process. I think it should be named after what the cutscene is doing though, not what the player happened to choose before.

For declined, maybe "deku tree ask again" makes sense, he's repeating his question if you are ready to go inside.

And then yeah, "mouth opening" for when you say yes.